LoopKitUI/Models/ColorPalette.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ColorPalette.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-07-29.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct ColorPalette {
+    public var theColor: Color?
```

Comment:
I'm not sure I understand how this will be used. There is only one color here?

Reply:
Sorry, this is an old PR. Deleting.